tests/subyt/test_generator.py

```
@@ -10,17 +9,16 @@
 
 
 class AssertingSink(Sink):
-    def __init__(self, test):
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)